src/main/java/car/CarList.java

```diff
@@ -17,4 +17,20 @@ public static void addCar(Car car) {
         System.out.println(car.getModel() + " | " + car.getLicensePlateNumber()
                 + " | $" + car.getPrice());
     }
+
+    /**
+     * Lists all cars in the current array list.
+     */
+    public static void listCars(){
```

Comment:
maybe "printCarList" as the name? so people don't get confused it as getCarList()